### DISCUSSION TITLE
Support for group syntax

### DISCUSSION THREAD

**Discussion by searchableguy** [dcastil/tailwind-merge#58] [Ideas]
Many alternative implementation to tailwind like windicss and twind have group syntax for variants, utilities, etc. 

https://twind.dev/handbook/grouping-syntax.html#variant-grouping

https://windicss.org/features/variant-groups.html

It would be nice to have support for this.

**Comment by dcastil** [comment]
Hey, I like the idea of supporting the group syntax of Windi CSS and others! I converted this to a discussion because I want to see if there are more people who'd want to have support for a feature like that before I throw this in for everyone.

There is also a possibility to do this with a plugin – with some small changes in the tailwind-merge API here and there.
